dev-docs/modules/userId.md

```diff
@@ -186,6 +186,49 @@ gulp build --modules=admixerIdSystem
    });
 {% endhighlight %}
 
+### Adtelligent
+
+The [Adtelligent](https://adtelligent.com) ID system is uniq per-session user identifier for providing high quality DMP data for advertisers
+
+Add it to your Prebid.js package with:
+
+{: .alert.alert-info :}
+gulp build --modules=userId,adtelligentIdSystem
+
+#### Adtelligent Configuration
+
+adtelligentIdSystem doesn't require any configuration and storage params, however, to it is asynchronous and to achieve better performance, it's recommended to store it for a short time period and configure refresh.
```

Comment:
Change:

adtelligentIdSystem doesn't require any configuration and storage params, however, to it is asynchronous and to achieve better performance, it's recommended to store it for a short time period and configure refresh.

to: 

adtelligentIdSystem adapter doesn't require any configuration or storage params. The adapter performs asynchronously and to achieve better performance it is recommended to set the `storage` object `refreshInSeconds` to a short period, such as ten minutes. At the end of the set storage refresh the adapter will refresh its configuration.